dockerfiles/compile.go

```diff
@@ -201,6 +201,18 @@ RUN ln -fs /usr/lib/systemd /lib/systemd` + installJava + installCMake,
 		tar_distro_name:   "ubuntu-noble",
 		package_extension: "deb",
 	},
+	{
+		from_image:  "ubuntu:oracular",
+		target_name: "oracular",
+		install_packages: `RUN set -x; apt-get update && \
+		DEBIAN_FRONTEND=noninteractive apt-get -y install git systemd \
+		autoconf libtool libcurl4-openssl-dev libltdl-dev libssl-dev libyajl-dev \
+		build-essential cmake bison flex file libsystemd-dev \
```

Comment:
Need to add `systemd-dev` and `debhelper` here. It is also need a newer version of fluent-bit. I tried 3.1.9 and it works. Created https://github.com/GoogleCloudPlatform/ops-agent/pull/1819 for the fluent-bit update.

Reply:
No need to update fluent-bit, the current version in the repo 3.1.6 builds correctly in Ubuntu Oracular.